client/src/ProtectedRoute.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+import Navbar from "./components/layout/Navbar"
+
+const ProtectedRoute = ({ isAuthenticated: isAuthenticated, component: Component, ...rest }) => {
```

Comment:
```suggestion
const ProtectedRoute = ({ isAuthenticated, component: Component, ...rest }) => {
```
You don't need to rename the prop here since it's the same name

Reply:
Whoops! Thank you.